dbld/packages.manifest

```diff
@@ -173,14 +173,14 @@ lcov                            [devshell]
 libc6-dbg                       [devshell]
 libglib2.0-0-dbgsym             [devshell]
 libjemalloc-dev                 [devshell]
-libjemalloc1-dbgsym             [devshell]
-libpcre2-dbg                    [devshell]
-libssl1.0.0-dbg                 [devshell]
+libjemalloc2-dbgsym             [devshell]
+libssl1.1-dbgsym                [devshell]
 linux-tools-generic             [devshell]
 locales                         [devshell]
 lsof                            [devshell]
 netcat                          [devshell]
-python-dbg                      [devshell]
+python2-dev			[devshell]
+python2-dbg                     [devshell]
```

Comment:
I'm running my "final tests" on this PR. `python2-pip` is missing from the devshell image:
```
+ install_pip2_packages
+ python2 -m pip install --upgrade pip
/usr/bin/python2: No module named pip
```

python2-pip is not part of the main focal repo, we'll need the `universe` repo to install `python-pip`.

Reply:
@bazsi It turned out that pip2 is not even available in the universe repo anymore.
I have a patch that installs pip2 using get-pip (@alltilla thanks for the hint).

https://github.com/MrAnno/syslog-ng/commit/f4bd74fe7e20cd7124164f0ddad8ea65cc69d8da

Everything else seems to be working now.

Reply:
I think this could go in after fixing the pip2 issue. Is MrAnno/syslog-ng@f4bd74f acceptable for devshell?